engine/pom.xml

```diff
@@ -229,10 +229,10 @@
                             javax.persistence.*;version="[2.0.0,3.0.0)";resolution:=optional,
                             javax.validation.*;version="[1.0.0,2.0.0)",
                             javax.xml.*;version="0",
-                            javax.el.*;version="[2.0.0,4.0.0)",
+                            javax.el.*;version="[2.0.0,4.0.0)";resolution:=optional,
                             org.xml.sax.*;version="0",
                             org.jboss.logging.*;version="[3.1.0,4.0.0)",
-                            com.fasterxml.classmate.*;version="0.8.0",
+                            com.fasterxml.classmate.*;version="[${classmate.version},2.0.0)",
```

Comment:
For a user intervals are nicer as it allows to work several clients with one and the same version of this dependency. Of course it depends on whether they evolve the API compatibly or not. I don't have a strong preference, only we should do it for other deps in the same way I'd say (see logging).